src/main/java/core/basesyntax/DrawFigureInfo.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface DrawFigureInfo {
```

Comment:
remove this interface, it's redundant to create a such interface with a such method

Reply:
Yeap, I agree, like a public interface AreaCalculator   - it`s doing nothing, I don`t see the need of interfaces in this task.